code/game/objects/structures/crates_lockers/closets/misc_vr.dm

```diff
@@ -93,7 +93,7 @@
  */
 /obj/structure/closet/secure_closet/pathfinder
```

Comment:
A request: Remove the cataloguers from these lockers and instead put them all in the explorer's weapon cabinet. Generally I would say there should be 1x Pathfinder cataloguer, 4x normal cataloguer, 4x compact cataloguer? Expeditions larger than 9 people are exceedingly rare anyway.

Reply:
Update: I would probably say that the pathfinder's special toys should be moved there as well. We do not really need a special pathfinder locker.